legacy/routes/queries.js

```diff
@@ -153,20 +153,20 @@ var queries = function (coverage) {
         utils.send400(res, next, 'Failed csv array scenario format "' + format + '", scenario "' + scenario + '"');
       }
     } else if (format === 'multi') {
-        console.log("In multi test\n");
-        if (scenario === 'null' && Object.keys(req.query).length == 0) {
-            coverage['UrlQueriesArrayMultiNull']++;
-            res.status(200).end();
-        } else if (scenario === 'empty' && Object.keys(req.query).length == 1 && req.query.arrayQuery === '') {
+      console.log("In multi test\n");
+      if (scenario === 'null' && Object.keys(req.query).length == 0) {
+        coverage['UrlQueriesArrayMultiNull']++;
+        res.status(200).end();
+      } else if (scenario === 'empty' && Object.keys(req.query).length == 0) {
         coverage['UrlQueriesArrayMultiEmpty']++;
         res.status(200).end();

```

Comment:
@lmazuel Is it correct? It appears that `/null` and `/empty` scenario now become same (with `query.length == 0`)?

Currently java generated code sends
`/queries/array/multi/string/null` for `/null`
and
`/queries/array/multi/string/empty?arrayQuery=` for `/empty`.

Just want to check what is the expected behavior.

Reply:
OK. I see. It is multi, not csv. Then it make sense.

Strange is that request `/queries/array/multi/string/valid?arrayQuery=ArrayQuery1%2Cbegin%21%2A%27%28%29%3B%3A%40%20%26%3D%2B%24%2C/?%23%5B%5Dend%2C%2C` still passes `/valid` case.